README.md

```diff
@@ -3,3 +3,4 @@
 Student: Alina
 Mentor: Dmitrii
 BRO Academy
+I am learning new things and I love it!
```

Comment:
Alina, could you please commit something else?)

Reply:
Done, please check :-)